applications/CompressiblePotentialFlowApplication/custom_elements/transonic_perturbation_potential_flow_element.cpp

```diff
@@ -379,6 +411,21 @@ void TransonicPerturbationPotentialFlowElement<TDim, TNumNodes>::GetWakeDistance
     noalias(distances) = GetValue(WAKE_ELEMENTAL_DISTANCES);
 }
 
+template <int TDim, int TNumNodes>
+void TransonicPerturbationPotentialFlowElement<TDim, TNumNodes>::GetEquationIdVectorExtendedElement(
+    EquationIdVectorType& rResult) const
+{
+    const int additional_node_id = GetAdditionalNode();
+
+    const auto& r_geometry = this->GetGeometry();
+    for (unsigned int i = 0; i < TNumNodes; i++)
+    {
+        rResult[i] = r_geometry[i].GetDof(VELOCITY_POTENTIAL).EquationId();
+    }
+    const auto& r_upwind_geometry = mpUpwindElement->GetGeometry();
```

Comment:
It is safer to access the upwind element with the function you implemented in the last PR: pGetUpwindElement, since it has a check in case for some reason there element doesn't have upwind element.